pkg/util/namer/namer.go

```diff
@@ -13,23 +13,27 @@ import (
 // an 8-character hash of the [base]-[suffix] string.  If the suffix is not too long,
 // it will truncate the base, add the hash of the base and return [base]-[hash]-[suffix]
 func GetName(base, suffix string, maxLength int) string {
+	if maxLength <= 0 {
```

Comment:
If we don't do this check, code later will compute lengths and slice indices incorrectly for certain inputs.